setup_data_scheme.md

```diff
@@ -486,3 +626,35 @@ example.prototype.validatorSchema = {
   }
 }
 ```
+## Data Loader
+For `readOne<model>` API, data loader would be used by default. It means that we can collect IDs, then fetch multiple records within one query. Hence, the read performance could be improved in Zendro.
+
+Here is an example for fetching multiple records within one request:
+
+```
+{
+  n0: readOneAccession(accession_id: "a-instance1") {
+    accession_id
+    collectors_name
+    location(search:null){
+      locationId
+    }
+  }
+  n1: readOneAccession(accession_id: "b-instance1") {
+    accession_id
+    collectors_name
+    location(search:null){
+      locationId
+    }
+  }
+  n2: readOneAccession(accession_id: "c-instance1") {
+    accession_id
+    collectors_name
+    location(search:null){
+      locationId
+    }
+  }
+}
+```
+
+For associated API like `location`, the root resolver would be invoked automatically. In specifically, `readOneAccession` would use the data loader in `accession` model. And the `location` API would use the data loader in `location` model. Hence, by using data loader optimization, only two queries would be executed instead of six queries.
```

Comment:
In the above example querying the associated location for each `readOneAccession` query invokes the location root resolver to search for associated records. By using the data loader we can collect all `readById` requests to the `accession` and the `location` model and optimize the queries to fetch those Ids together. This improves the amount of executed queries from six to two.